kik_unofficial/client.py

```diff
@@ -45,7 +44,15 @@ def __init__(self, callback: callbacks.KikClientCallback, kik_username, kik_pass
         :param kik_password: the kik password to log in with.
         :param kik_node: the username plus 3 letters after the "_" and before the "@" in the JID. If you know it,
                          authentication will happen faster and without a login. otherwise supply None.
-        """
+        :param device_id: a unique device ID. If you don't supply one, a random one will be generated.
+        :param android_id: a unique android ID. If you don't supply one, a random one will be generated.
+        :param logging: If true, turns on logging to stdout (default: False)
+        """
+        # turn on logging with basic configuration
+        if logger:
+            from kik_unofficial.utilities.logging_utils import set_up_basic_logging 
```

Comment:
Please move the import to the top and call `logging_utils.set_up_basic_logging`

Reply:
That would cause a circular import since logging_utils imports kik_unofficial

Reply:
Oh, I see. So let's move `KikClient.log_format` to `logging_utils`.